kong/db/schema/topological_sort.lua

```diff
@@ -1,8 +1,7 @@
 local constants = require "kong.constants"
-local utils = require "kong.tools.utils"
+local topological_sort  = require("kong.db.sort").topological_sort
```

Comment:
If we have `kong/db/utils.lua`, then this line could be 
```
local utils = require "kong.db.utils"
```
and we will not change other lines.